### DISCUSSION TITLE
how to connect to Influxdb on Redash

### DISCUSSION THREAD

**Discussion by sherrysong2000** [getredash/redash#5918] [Ideas]
i keep receiving errors either not supported scheme

**Comment by arikfr** [comment]
It's difficult to provide a solution without knowing the full error message you're receiving.

**Comment by fancyremarker** [comment]
I'm hitting what might be the same issue when attempting to connect to an InfluxDB (v1.4.2) database from Redash, over the InfluxDB's HTTPS connection. When I click "Test Connection" on such a newly created Data Source, I get:

```
Connection Test Failed:
Unknown scheme "https".
```

We have InfluxDB configured with `https-enabled = true`. Does Redash only support connecting to InfluxDB over plain HTTP?
